src/main/java/jenkins/plugins/slack/user/SlackUserProperty.java

```diff
@@ -38,16 +39,17 @@
     private boolean disableNotifications;
 
     @DataBoundConstructor
-    public SlackUserProperty(String userId, boolean disableNotifications) {
-        this.userId = userId;
-        this.disableNotifications = disableNotifications;
+    public SlackUserProperty() {
+        this.userId = null;
```

Comment:
do we need any content in this constructor at all?

Reply:
No, we don't